main.tf

```diff
@@ -8,7 +8,13 @@ resource "azurerm_resource_group" "default" {
   name     = var.namespace
   location = var.location
 
-  tags = var.tags
+  tags = merge(
```

Comment:
We could merge the tags into a local variable and use that instead of merging everywhere in this file.